grease/src/Grease/Refine.hs

```diff
@@ -568,6 +555,81 @@ execAndRefine bak solver _fm la memVar anns heuristics argNames argShapes initSt
       ProveCantRefine (Timeout{}) -> "symex timeout"
       ProveCantRefine (Unsupported{}) -> "unsupported feature"
 
+-- TODO: Fold into `refinementLoop`?
+refineOnce ::
```

Comment:
Id probably tack on a doc here since it is exported and clients need to know to pass it to the loop

Reply:
Whoops, yep!